oak_launcher_utils/src/launcher/virtualized.rs

```diff
@@ -100,7 +100,8 @@ impl Instance {
         // Needed to expose advanced CPU features. Specifically RDRAND which is required for remote
         // attestation.
         cmd.args(["-cpu", "IvyBridge-IBRS,enforce"]);
-
+        // TODO(#3786): Make memory configurable.
+        cmd.args(["-m", "4G"]);
```

Comment:
This seems a lot, since each test instance will have to allocate all this memory I guess? What was the previous (default) value? It would be nicer to be more conservative with the increase if possible.